### DISCUSSION TITLE
[ERROR] Error: Please install the package(s) for your selected database type: "sqlite3"

### DISCUSSION THREAD

**Discussion by DeinVorbild** [discord-tickets/bot#285] [Support (Q&A)]
How i can fix it ?

https://prnt.sc/W2pWb9nr3xXQ

**Comment by RooRay** [comment]
> How i can fix it ?
> 
> https://prnt.sc/W2pWb9nr3xXQ

Just install sqlite3 using NPM with `npm install sqlite3`

**Comment by DeinVorbild** [comment]
Not work :c

https://prnt.sc/Alw0QJWH5e-7

https://prnt.sc/827hOHoYuHcC
